tests/metrics/test_classification.py

```diff
@@ -53,6 +53,20 @@ def test_confusion_matrix(normalize, num_classes):
     assert isinstance(cm, torch.Tensor)
 
 
+@pytest.mark.parametrize(['normalize', 'num_classes'], [
+    pytest.param(True, 3)
+])
+def test_confusion_matrix_norm(normalize, num_classes):
```

Comment:
Add small description here what the test does, something like
```suggestion
def test_confusion_matrix_norm(normalize, num_classes):
    """ test that user is warned if confusion matrix contains nans that are changed to zeros"""
```